head/src/main/kotlin/io/qalipsis/core/head/campaign/scheduler/MonthlyScheduling.kt

```diff
@@ -0,0 +1,69 @@
+package io.qalipsis.core.head.campaign.scheduler
+
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.inject.Singleton
+import java.time.Instant
+import java.time.ZoneId
+import java.time.temporal.ChronoUnit
+import javax.validation.constraints.NotBlank
+
+/**
+ * Monthly implementation of [Scheduling].
+ *
+ * @author Joël Valère
+ */
+
+@Singleton
+internal class MonthlyScheduling(
+    @field:Schema(
+        description = "The time zone ID to use to schedule the test",
+        required = true,
+        example = "Africa/Douala"
+    )
+    @field:NotBlank
+    private val timeZone: String,
+    @field:Schema(
+        description = "The set of -15 to 31-based integers that restrict the hours in the day, when the campaign can be executed ; for negative values, the restriction is calculated from the 1st of the next month otherwise, 1 stands for the first of the month and 31 for the thirty-first of the month",
```

Comment:
```suggestion
        description = "The set of -15 to 31-based integers that restrict the hours in the day, when the campaign can be executed; for negative values, the restriction is calculated from the 1st of the next month; otherwise, 1 stands for the first day of the month and 31 for the thirty-first day of the month, if there is one",
```